tests/suite/test_app_protect_grpc.py

```diff
@@ -0,0 +1,168 @@
+import requests
+import subprocess
+import pytest
+
+from settings import TEST_DATA, DEPLOYMENTS
+from suite.custom_resources_utils import (
+    create_ap_logconf_from_yaml,
+    create_ap_policy_from_yaml,
+    delete_ap_policy,
+    delete_ap_logconf,
+)
+from suite.resources_utils import (
+    wait_before_test,
+    create_example_app,
+    wait_until_all_pods_are_ready,
+    create_items_from_yaml,
+    delete_items_from_yaml,
+    delete_common_app,
+    replace_configmap_from_yaml,
+    create_ingress_with_ap_annotations,
+    wait_before_test,
+    get_file_contents,
+)
+from suite.yaml_utils import get_first_ingress_host_from_yaml
+
+
+ap_policies_under_test = ["grpc-block-saygoodbye", "grpc-bloc-sayhello"]
```

Comment:
this variable isn't used. is it intentional?

Reply:
this is redundant and will be removed, I added it in the beginning to parameterize the single test based on different policies but the code was turning out to be more convoluted and reduction in line of code and 40sec savings didn't seemed worth it.